docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,382 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
+
+Default `generic.recept.json`:
+
+```
+  {
+      "stacks": {
+          "dump_stacks": true,
+          "first_thread_only": false,
+          "max_stack_size": 0
+      },
+      "maps": {
+          "dump_by_name": [
+              "[vdso]"
+          ]
+      },
+      "compression": {
+          "compressor": "",
+          "extension": "",
+          "in_tar": true
+      },
+      "dump_auxv_so_list": true,
+      "dump_pthread_list": true,
+      "dump_robust_mutex_list": true,
+      "dump_scope": 1024,
+      "dump_build_id": true,
+      "live_dumper": true,
+      "write_proc_info": true,
+      "proc_info_exclude": [
+          "smaps"
+      ],
+      "write_debug_log": false,
+      "dump_fat_core": false,
+      "dump_pointed_by_regs": true
+  }
+```
+
+### dump_systemstate
+
+**dump_systemstate** is used to dump informations that can be useful for developers to fix bugs in the crashed applications.
+
+Default dump contains:
+-   Basic information:
+    -   Binary version from `/etc/info.ini`
+    -   Tizen version from `/etc/tizen-release`
+    -   Kernel version from `/proc/version`
+    -   Boot arguments from `/proc/cmdline`
+    -   CPU & system architecture from `/proc/cpuinfo`
+    -   System uptime from `/proc/uptime`
+    -   Local time from `/opt/etc/localtime`
+-   Resource usage information:
+    - System statistics from `/proc/stat`
+    - System memory usage from `/proc/meminfo`
+    - System disk I/O statistics from `/proc/diskstats`
+    - System disk space usage
+    - System memory statistics
+-   Process information:
+    - Process information by `ps` command
+-   Device information:
+    - Device major numbers from `/proc/devices` 
+
+
+The dump contents of **dump_systemstate** can be modified by the following command line options:
+
+Option | Description
+-------|----------------------------------
+-f     | write to file (instead of stdout)
+-k     | dump kernel messages (root only)
+-d     | dump dlog messages
+-j     | dump journal log messages
+-p     | dump list of installed packages
+-e     | dump extras defined in the config
+
+Depending on the command line options, **dump_systemstate** can save the output of these commands:
+
+-   `/bin/df -h`
+-   `/bin/du -ah /opt --exclude=/opt/usr`
+-   `/bin/ls -al /opt/etc/localtime`
+-   `/bin/top -bcH -n 1`
+-   `/bin/ps auxfw`
+-   `/bin/memps -v`
+-   `/bin/buxton2ctl dump memory`
+-   `/bin/buxton2ctl dump system`
+-   `/usr/bin/pkgcmd -l`
+-   `/bin/dmesg -T`
+-   `/bin/dlogutil -d -v threadtime -u 16384`
+-   `/bin/journalctl -b -n 1024`
+
+
+There are two types of configuration files, `files/` subdirectory and `programs/` subdirectory. These files allow to specify the results of additional files and program that will be included in the **dump_systemstate** output:
+
+-   `files/` subdirectory: Specified files will be appended to the **dump_systemstate** output. File contains:
```

Comment:
@mijin85cho, in my opinion, I think "File contains:" would be better. As we are specifying what does the file that is appended to the dump_systemstate output contain.

Please correct me if my understanding is not right.

Reply:
Then would you please figure out other way, not "File contains:" to avoid two colons in one sentence? For here we have bunch of colons now..
I hope you search "two colons in one sentence" in Google.

Reply:
@mijin85cho,

As suggested by you, can we rephrase as:
`files/` subdirectory: Specified files will be appended to the **dump_systemstate** output.

Reply:
@mijin85cho, thank you for your comment.

@mmoscicki2, please revert back this sentence.